content/css/jenkins.css

```diff
@@ -826,6 +834,11 @@ z.f9f9f9.section .ji-blog-list>.post>.body>.teaser:after,
   border-bottom: 1px solid #ccc
 }
 
+.blog-posts > h3 {
+  color: black;
+  font-weight: 900;
+}
```

Comment:
Before:
<img width="743" alt="Screenshot 2022-08-08 at 12 41 36" src="https://user-images.githubusercontent.com/1831569/183400380-0b61b720-337e-42f9-b566-3beb81201caf.png">

After:
<img width="754" alt="Screenshot 2022-08-08 at 12 41 42" src="https://user-images.githubusercontent.com/1831569/183400394-ec1ded1a-1e5f-4e2e-a33a-5e5ed1606997.png">

Why? And what about this is a "correction"?